docs/_static/auth/iam.yaml

```diff
@@ -145,8 +145,17 @@ paths:
               schema:
                 type: string
                 format: uri
-  /oauth2/request.jwt/{id}:
+  /oauth2/{did}/request.jwt/{id}:
```

Comment:
technically we don't need the DID in the path, but all other paths have it...

Reply:
The same applies to a OpenID4VCI endpoint, so I don't think it needs to be per se. One more thing to validate...